vanilla-tests/src/test/java/fixtures/nonstringenum/NonStringEnumTests.java

```diff
@@ -27,6 +27,7 @@ public void getInt() {
         assertEquals(IntEnum.FOUR_HUNDRED_TWENTY_NINE, actual);
     }
 
+    @Disabled("wire type changed for Swagger extensible non-string enum")
```

Comment:
Actually seems all tests in this class are disabled. Shall we delete this test class?

Reply:
Leave it. It is autorest